js/views/modals/Settings/advanced/Advanced.js

```diff
@@ -242,7 +243,6 @@ export default class extends baseVw {
   }
 
   render() {
-    super.render();
```

Comment:
This render should probably be up top. It's on modals that it needs to be after the content is added to `this.el`.